pkg/resolve/selector.go

```diff
@@ -47,6 +47,11 @@ func MatchesSelector(doc *yaml.Node, selector labels.Selector) (bool, error) {
 }
 
 func docKind(doc *yaml.Node) (string, error) {
+	// Null nodes will fail the check below, so simply ignore them.
+	if doc.Tag == "!!null" {
```

Comment:
Where does `!!nulll` come from? Does that yaml package export something we can use instead? I would never be able to come up with this string on my own :P

Reply:
I believe it is the canonical null type in YAML, similar to !!str or !!int. See https://yaml.org/type/null.html.